Replicator/Pusher+Revs.cc

```diff
@@ -106,21 +107,43 @@ namespace litecore::repl {
             }
         }
 
-        auto fullRevID = alloc_slice(_db->convertVersionToAbsolute(request->revID));
+        auto            fullRevID    = alloc_slice(_db->convertVersionToAbsolute(request->revID));
+        alloc_slice     currentRevID = fullRevID;
+        constexpr slice vvSeparator  = ",;"_sl;
+        if ( currentRevID.findAnyByteOf(vvSeparator) ) {
+            try {
+                VersionVector vv = VersionVector::fromASCII(currentRevID);
+                currentRevID     = vv.current().asASCII();
+            } catch ( ... ) {
+                // it's not a VV even with comma or semi-colon in it.
+                throw;
+            }
```

Comment:
This try-catch is pointless; it doesn't do anything.

Reply:
Right, I was thinking to add some logs there. Perhaps not that useful; hopefully we will get stacktrace.